ssm-diff.py

```diff
@@ -36,15 +36,18 @@ def plan(filename):
     parser.add_argument('-f', help='local state yml file', action='store', dest='filename', nargs=1, default='parameters.yml')
     subparsers = parser.add_subparsers(help='commands')
 
-    parser_init = subparsers.add_parser('plan', help='display changes between local and remote states')
-    parser_init.set_defaults(func=plan)
+    parser_plan = subparsers.add_parser('plan', help='display changes between local and remote states')
+    parser_plan.set_defaults(func=plan)
+    parser_plan.add_argument('--paths', nargs='+', default=['/'], help='filter SSM paths')
```

Comment:
Can you please move "paths" parameter to a parent parser, to avoid duplication?
Also, a bit more verbose help would be nice